qiskit/providers/models/backendconfiguration.py

```diff
@@ -400,17 +410,80 @@ def acquire(self, qubit: int) -> AcquireChannel:
             raise BackendConfigurationError("Invalid index for {}-qubit systems.".format(qubit))
         return AcquireChannel(qubit)
 
-    def control(self, channel: int) -> ControlChannel:
+    def control(self, channel: int = None,
```

Comment:
The sig here should be `def control(qubits: Iterable, channel: int = None) -> ControlChannel:`

Reply:
fixed in [c2649b4](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/c2649b40cfe0a9679661644414537f7621529ed2)